tests/Integration/Behaviour/Features/Scenario/CLDR/cldr.feature

```diff
@@ -0,0 +1,58 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s cldr
+@reset-database-before-feature
+Feature: CLDR display for prices
+
+  Background:
+    Given shop "shop1" with name "test_shop" exists
+
+  Scenario: Display EUR
+    When I add a new language "language1" with following properties:
+      | name             | English (US) |
+      | iso_code         | en           |
+      | language_code    | en-us        |
+      | locale           | en-US        |
+      | date_format_lite | m/d/Y        |
+      | date_format_full | m/d/Y H:i:s  |
+      | is_rtl           | 0            |
+    And I add a new language "language2" with following properties:
```

Comment:
It feels like you are testing **CR**UD operations here, which should be tested somewhere else. What if we had some shortuct `Given`s here? For example:

`Given There is "English" language "lang1"`
`And There is "French" language "lang2"`

Or something similar, so this Behat scenario is focused in testing price display and not CRUD actions, wdyt?

Same for currency.

Reply:
Yes I added these statements in behat contexts to allow adding a full Language or Currency, I'm not fully used to using behat yet so I'm not sure I understand what you mean by using a `Given` shortcut Do you mean matching the "English" word to adding a Language with hard coded values matching the english language?

What I had in mind when I used/added these statements (from develop branch) was that I didn't want to multiply the entry points to add these fixtures for several reasons:
- to avoid duplicate/multiply the way we add fixtures for the tests
- we have many bug on the CLDR linked to the way the currency are added in database, so I wanted the behat context to be the closest to the real use case
- in 177 some additional tests will need to be added to test customization so these statements will be useful to test this behaviour

That said maybe it is contradictory to the behat way of adding fixtures, I totally understand your aim to focus on the display testing but how could we do it and still meet the other constraints I mentioned?

Reply:
CRUD operations are meaningful in a BO, because when we click on this "submit" button there is a modification of the system and the domain. Even if it's a basic modification (it sets the data) it's worth being tested if in the past it has proven to be unstable 👍